tests/draft2019-09/propertyNames.json

```diff
@@ -43,6 +43,35 @@
             }
         ]
     },
+    {
+        "description": "propertyNames validation with pattern for camelCase",
```

Comment:
Can you remove the reference to camel case in the test names since it seems we managed to simplify things?

After that yeah this looks great!